docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2039 @@
+{
```

Comment:
<p>I think it's less confusing to call it "samples from a vector of two normal distributions", and label <code>z[0]</code> and <code>z[1]</code>. They are indeed equivalent to a specific mvnormal but I think that distracts from the goal here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

Reply:
<p>+1. Changed to "Samples Histogram"</p>

Reply:
+1. Changed to "Samples Histogram"